packages/core/src/amazonq/util/files.ts

```diff
@@ -186,7 +187,7 @@ export async function prepareRepoData(
     } catch (error) {
         getLogger().debug(`featureDev: Failed to prepare repo: ${error}`)
         if (error instanceof ToolkitError && error.code === 'ContentLengthError') {
-            throw new ContentLengthError()
+            throw new CommonAmazonQContentLengthError(error.message)
```

Comment:
You don't need to pass in the error message if you don't end up using it anywhere